docs/Coiote_IoT_DM/Coiote_videos/How_to_remotely_configure_IoT_devices.md

```diff
@@ -0,0 +1,17 @@
+# How to remotely configure IoT devices?
+
+With IoT devices enabled with LwM2M protocol, you can use a set of standardized remote operations to configure your IoT devices remotely.
+
+To prepare your device with LwM2M connectivity, grab Anjay IoT SDK, Zephyr Anjay Client, or any LwM2M Client. Check out more handy links below.
```

Comment:
Same as above.

Also `Check out more handy links below` but there is no link here, so definitely not `more handy links`.